### PR TITLE
fix CLI and etcd tests to work with the new format of oauth{access,authorize} tokens

### PR DESCRIPTION
**Pull request by stlaz**
/assign @sttts 
cc @EmilyM1

### CODE REVIEW

File: test/extended/util/client.go [hunk 1]
```diff
@@ -749,11 +745,22 @@ func (c *CLI) GetClientConfigForUser(username string) *rest.Config {
...
+	userClientConfig.BearerToken = privToken
 
 	return userClientConfig
 }
 
+// GenerateTokenPair returns two tokens to use with OpenShift OAuth-based authentication.
+// The first token is a private token meant to be used as a Bearer token to send
+// queries to the API, the second token is a hashed token meant to be stored in
+// the database.
+func GenerateTokenPair() (privToken, pubToken string) {
```
Comment (sttts) [inline test/extended/util/client.go:757]: nit: GenerateOauthTokenPair

**Review by EmilyM1 — APPROVED** [review]
/lgtm
/approve

**Comment by smarterclayton** [comment]
Overriding failing jobs

**Pull request merged**